src/lib/store/index.js

```diff
@@ -16,26 +16,15 @@ const config = {
   whitelist: persistWhitelist
 }
 
-// copied and changed from https://github.com/LogRocket/redux-logger/blob/3ca9f2c1ecf17a7acf18c6fa0bbf4b6b239738f1/src/core.js#L25
-function barTitleFormatter(action, time) {
-  const parts = ['[COZY-BAR]']
-
-  parts.push(`${String(action.type)}`)
-  if (time) parts.push(`@ ${time}`)
-
-  return parts.join(' ')
-}
-
 const loggerMiddleware = createLogger({
-  titleFormatter: barTitleFormatter,
```

Comment:
I though you added support for group/end https://github.com/cozy/minilog/commit/80414d3d28b609c7452c937524c1838928dc6a06

Reply:
Ah yes, but group and groupEnd are always shown, without consideration for the current log level (since the group call does not know about the message level), which pollutes the console.